Source/Application/PolytempoNetwork/Polytempo_NetworkApplication.cpp

```diff
@@ -324,6 +324,12 @@ void Polytempo_NetworkApplication::openScoreFile(File aFile)
         return;
     }
 
+    if (score && !score->isReady())
+    {
+        DBG("Can't open file, still loading the previous one...");
```

Comment:
Müsste das nicht auch dem User in irgend einer Form zurückgemeldet werden?

Reply:
Habe mir das auch überlegt. Im Moment bekommt der User diese Information bloss implizit: Da während des Ladens eine Box mit "loading..." auf dem Bildschirm angezeigt wird, muss sich der User zusammenreimen, dass das Programm gerade mit einer anderen Aufgabe blockiert ist.

![IMG_0557](https://user-images.githubusercontent.com/1252342/129614750-535144b3-cd1c-43c1-b5bf-b6840fe36794.png)

Ich weiss, dass das nicht 100% elegant ist, aber für mich reicht es im Moment. Insbesondere, weil der Ladeprozess meist weniger als 1s dauert. Nur bei dem Stück, das gerade geprobt wird, werden Hunderte von Bildern geladen (ohne Witz!), und das dauert dann etwa eine halbe Minute.

Ich habe aber sowieso vor, bei einer nächsten Feature-Runde mal die Alert-Boxen zu überarbeiten und in eine Art Konsole-Fenster (ähnlich wie bei Max) umzuwandeln. Es gibt nämlich noch viel mehr, was dem User mitgeteilt werden müsste (z.B. "unbekannte" OSC-Messages usw.)